packages/fxa-auth-server/config/index.ts

```diff
@@ -2097,15 +2097,9 @@ const convictConf = convict({
         env: 'CMS_WEBHOOK_CACHE_INVALIDATION_ENABLED',
         format: Boolean,
       },
-      headerKey: {
-        default: 'fxa-cms-cache-webhook-auth',
-        doc: 'The header key where to location a value to auth the request',
-        env: 'CMS_WEBHOOK_CACHE_INVALIDATION_HEADER',
-        format: String,
-      },
-      headerVal: {
-        default: 'CHANGEME',
-        doc: 'The header key where to location a value to auth the request',
+      secret: {
+        default: 'Bearer CHANGEME',
+        doc: 'The secret used in the Authorization header for the webhook',
```

Comment:
So Strapi offers two ways of doing this one thing? :-/

Reply:
Oh, strapi does not technically say you have to use the `authorization` header. This was to standarized how we did webhook auth. The cache invalidation (header/val) vs l10n webhook (authorization/secret). 

I don't have a strong opinion either, the header/val is more flexible.